### PR TITLE
Updating wavedrom

### PR DESCRIPTION
**Pull request by kersten1**
https://github.com/riscv/riscv-isa-manual/issues/1335

### CODE REVIEW

File: src/images/wavedrom/reg-based-ldnstr.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op', attr: ['2', 'C0', 'C0', 'C0', 'C0', 'C0'], type: 8},
-  {bits: 3, name: 'rd`',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
+  {bits: 3, name: 'rd',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
   {bits: 2, name: 'imm', attr:['2', 'offset[2|6]', 'offset[7:6]', 'offset[7:6]', 'offset[2|6]', 'offset[7:6]'], type: 2},
-  {bits: 3, name: 'rs1`', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},
+  {bits: 3, name: 'rs1', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},
```
Comment (ved-rivos) [inline src/images/wavedrom/reg-based-ldnstr.adoc:10]: The prime should be retained.. ` should have been changed to '

Comment (kersten1) [reply src/images/wavedrom/reg-based-ldnstr.adoc:10]: Ah so it looks like `{bits: 3, name: 'rs1'', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},` ?

Is this true for all of them?

Comment (ved-rivos) [reply src/images/wavedrom/reg-based-ldnstr.adoc:10]: Yes, all instances where `rs1` or `rs2` or `rd` is a 3-bit field then its marked with ' to indicate that it holds the restricted set of registers (as compared to when they are 5-bit fields..

Comment (ved-rivos) [reply src/images/wavedrom/reg-based-ldnstr.adoc:10]: The #1335 reported four typos where ` was used instead of '.

**Comment by kersten1** [comment]
looked in the old manual and it appears to be an artifact of the conversion to asciidoc. Need some sort of math symbol. I'll do some research and see if I can replicate.

Screenshot 2024-06-06 at 2 22 38 PM

**Comment by wmat** [comment]
There are places that require the prime mark, the original latex spec will
have the correct text. This may be a font issue as some don’t have very
good math symbol support.

Bill Traynor
Documentation Build and Release Engineer
RISC-V International

Join us in Munich, Germany at RISC-V Summit Europe
<https://riscv-europe.org/summit/2024/> from 24-28 June. Be a part of the
new wave of European computing innovation!

File: src/images/wavedrom/c-cs-format-ls.adoc [hunk 2]
```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8, attr: ['2', 'C0','C0','C0','C0','C0']},
-  {bits: 3, name: 'rs2`',   type: 3, attr: ['3', 'src','src','src','src','src']},
+  {bits: 3, name: 'rs2',   type: 3, attr: ['3', 'src','src','src','src','src']},
```
Comment (kersten1) [inline src/images/wavedrom/c-cs-format-ls.adoc:8]:
```suggestion
  {bits: 3, name: 'rs2&prime;',   type: 3, attr: ['3', 'src','src','src','src','src']},
```

File: src/images/wavedrom/c-cs-format-ls.adoc [hunk 3]
```diff
@@ -5,9 +5,9 @@
...
   {bits: 2, name: 'imm',    type: 2, attr: ['2', 'offset[2|6]','offset[7:6]','offset[7:6]','offset[2|6]','offset[7:6]']},
-  {bits: 3, name: 'rs1`',   type: 3, attr: ['3', 'base','base','base','base','base']},
+  {bits: 3, name: 'rs1',   type: 3, attr: ['3', 'base','base','base','base','base']},
```
Comment (kersten1) [inline src/images/wavedrom/c-cs-format-ls.adoc:10]:
```suggestion
  {bits: 3, name: 'rs1&prime;',   type: 3, attr: ['3', 'base','base','base','base','base']},
```

File: src/images/wavedrom/cr-register.adoc [hunk 4]
```diff
@@ -38,7 +38,7 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rd`',    type: 2},
+  {bits: 3, name: 'rd',    type: 2},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:41]:
```suggestion
  {bits: 3, name: 'rd&prime;',    type: 2},
```

File: src/images/wavedrom/cr-register.adoc [hunk 5]
```diff
@@ -48,9 +48,9 @@
...
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:51]:
```suggestion
  {bits: 3, name: 'rd&prime;',    type: 2},
```

File: src/images/wavedrom/cr-register.adoc [hunk 6]
```diff
@@ -48,9 +48,9 @@
...
   {bits: 2, name: 'imm',    type: 3},
-  {bits: 3, name: 'rs1`',   type: 4},
+  {bits: 3, name: 'rs1',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:53]:
```suggestion
  {bits: 3, name: 'rs1&prime;',   type: 4},
```

File: src/images/wavedrom/cr-register.adoc [hunk 7]
```diff
@@ -60,9 +60,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rs2`',   type: 4},
+  {bits: 3, name: 'rs2',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:63]:
```suggestion
  {bits: 3, name: 'rs2&prime;',   type: 4},
```

File: src/images/wavedrom/cr-register.adoc [hunk 8]
```diff
@@ -60,9 +60,9 @@
...
   {bits: 2, name: 'imm',    type: 3},
-  {bits: 3, name: 'rs1`',   type: 4},
+  {bits: 3, name: 'rs1',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:65]:
```suggestion
  {bits: 3, name: 'rs1&prime;',   type: 4},
```

File: src/images/wavedrom/cr-register.adoc [hunk 9]
```diff
@@ -72,9 +72,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rs2`',   type: 4},
+  {bits: 3, name: 'rs2',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:75]:
```suggestion
  {bits: 3, name: 'rs2&prime;',   type: 4},
```

File: src/images/wavedrom/cr-register.adoc [hunk 10]
```diff
@@ -72,9 +72,9 @@
...
   {bits: 2, name: 'funct2', type: 8},
-  {bits: 3, name: 'rd`/rs1`',   type: 7},
+  {bits: 3, name: 'rd`/rs1',   type: 7},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:77]:
```suggestion
  {bits: 3, name: 'rd`/rs1&prime;',   type: 7},
```

File: src/images/wavedrom/cr-register.adoc [hunk 11]
```diff
@@ -84,7 +84,7 @@
 {reg: [
   {bits: 2, name: 'op',     type: 8},
   {bits: 5, name: 'offset', type: 3},
-  {bits: 3, name: 'rd`/rs1`',   type: 7},
+  {bits: 3, name: 'rd`/rs1',   type: 7},
```
Comment (kersten1) [inline src/images/wavedrom/cr-register.adoc:87]:
```suggestion
  {bits: 3, name: 'rd`/rs1&prime;',   type: 7},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 12]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
   {bits: 3, name: 'funct3', type: 8},
 
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rd`',    type: 2},
+  {bits: 3, name: 'rd',    type: 2},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:22]:
```suggestion
  {bits: 3, name: 'rd&prime;',    type: 2},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 13]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 8, name: 'imm',    type: 3},
   {bits: 3, name: 'funct3', type: 8},
 
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rd`',    type: 2},
+  {bits: 3, name: 'rd',    type: 2},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:27]:
```suggestion
  {bits: 3, name: 'rd&prime;',    type: 2},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 14]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 2, name: 'imm',    type: 3},
-  {bits: 3, name: 'rs1`',   type: 4},
+  {bits: 3, name: 'rs1',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:29]:
```suggestion
  {bits: 3, name: 'rs1&prime;',   type: 4},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 15]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 3, name: 'imm',    type: 3},
   {bits: 3, name: 'funct3', type: 8},
 
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rs2`',   type: 4},
+  {bits: 3, name: 'rs2',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:34]:
```suggestion
  {bits: 3, name: 'rs2&prime;',   type: 4},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 16]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 2, name: 'imm',    type: 3},
-  {bits: 3, name: 'rs1`',   type: 4},
+  {bits: 3, name: 'rs1',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:36]:
```suggestion
  {bits: 3, name: 'rs1&prime;',   type: 4},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 17]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 3, name: 'imm',    type: 3},
   {bits: 3, name: 'funct3', type: 8},
 
   {bits: 2, name: 'op',     type: 8},
-  {bits: 3, name: 'rs2`',   type: 4},
+  {bits: 3, name: 'rs2',   type: 4},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:41]:
```suggestion
  {bits: 3, name: 'rs2&prime;',   type: 4},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 18]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 2, name: 'funct2', type: 8},
-  {bits: 3, name: 'rd` / rs1`',   type: 7},
+  {bits: 3, name: 'rd` / rs1',   type: 7},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:43]:
```suggestion
  {bits: 3, name: 'rd` / rs1&prime;',   type: 7},
```

File: src/images/wavedrom/cr-registers-new.adoc [hunk 19]
```diff
@@ -19,33 +19,33 @@ ${wd({reg: [
...
   {bits: 6, name: 'funct6', type: 8},
 
   {bits: 2, name: 'op',     type: 8},
   {bits: 5, name: 'offset', type: 3},
-  {bits: 3, name: 'rd` / rs1`',   type: 7},
+  {bits: 3, name: 'rd` / rs1',   type: 7},
```
Comment (kersten1) [inline src/images/wavedrom/cr-registers-new.adoc:48]:
```suggestion
  {bits: 3, name: 'rd` / rs1&prime;',   type: 7},
```

File: src/images/wavedrom/reg-based-ldnstr.adoc [hunk 20]
```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op', attr: ['2', 'C0', 'C0', 'C0', 'C0', 'C0'], type: 8},
-  {bits: 3, name: 'rd`',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
+  {bits: 3, name: 'rd',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
```
Comment (kersten1) [inline src/images/wavedrom/reg-based-ldnstr.adoc:8]:
```suggestion
  {bits: 3, name: 'rd&prime;',  attr: ['3', 'dest', 'dest','dest','dest','dest'], type: 3},
```

File: src/images/wavedrom/reg-based-ldnstr.adoc (hunk 1 above)
Comment (kersten1) [inline src/images/wavedrom/reg-based-ldnstr.adoc:10]:
```suggestion
  {bits: 3, name: 'rs1&prime;', attr: ['3', 'base', 'base', 'base', 'base', 'base'], type: 2},
```

**Comment by wmat** [comment]
Note that &prime; does not work in wavedrom files.  See:
Screenshot from 2024-06-07 13-16-03

File: src/images/wavedrom/c-cs-format-ls.adoc [hunk 21]
```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8, attr: ['2', 'C0','C0','C0','C0','C0']},
-  {bits: 3, name: 'rs2`',   type: 3, attr: ['3', 'src','src','src','src','src']},
+  {bits: 3, name: 'rs2&prime;',   type: 3, attr: ['3', 'src','src','src','src','src']},
```
Comment (kersten1) [inline src/images/wavedrom/c-cs-format-ls.adoc:8]:
```suggestion
  {bits: 3, name: 'rs2 /u2032',   type: 3, attr: ['3', 'src','src','src','src','src']},
```

File: src/images/wavedrom/c-cs-format-ls.adoc [hunk 22]
```diff
@@ -5,9 +5,9 @@
 ....
 {reg: [
   {bits: 2, name: 'op',     type: 8, attr: ['2', 'C0','C0','C0','C0','C0']},
-  {bits: 3, name: 'rs2`',   type: 3, attr: ['3', 'src','src','src','src','src']},
+  {bits: 3, name: 'rs2 /u2032',   type: 3, attr: ['3', 'src','src','src','src','src']},
```
Comment (kersten1) [inline src/images/wavedrom/c-cs-format-ls.adoc:8]:
```suggestion
  {bits: 3, name: 'rs2ʹ',   type: 3, attr: ['3', 'src','src','src','src','src']},
```

**Comment by kersten1** [comment]
We have a winner!

**Comment by wmat** [comment]
LGTM

**Pull request merged**